app/components/memorable_date_component.html.erb

```diff
@@ -20,7 +20,12 @@
             <%= p.input(
                   :month,
                   pattern: '(1[0-2])|(0?[1-9])',
-                  wrapper_html: { class: 'usa-form-group usa-form-group--month margin-bottom-0' },
+                  wrapper_html: {
+                    aria: {
+                      labelledby: [ "memorable-date-month-span-#{unique_id}" ],
```

Comment:
To add on to my previous comment, do we need a new element at all, or could we point this to the existing `<label>` element?

Reply:
good point, just updated this